resources/src/books.php

```diff
@@ -27,7 +26,7 @@ public function disp_book($recent)
             $stmt2 = $this->pdo->query('SELECT * FROM Book order by added_on desc limit 4');
          }
          $op=array();
-         while($row=$stmt2->fetch(PDO::FETCH_ASSOC))
+         while($row=$stmt2->fetch())
```

Comment:
Please verify once PDO::FETCH_ASSOC, You might need to pass this into fetch method.

Reply:
@guptahemant I have set it default in the config while declaring the PDO object in dbConn.php so it needed?